tests_profile

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# Prepare environment variables for building and running tests.
+# This requires executables and libraries to be able to find libjvm and Rust stdlib.
+#
+# ¡Keep it MacOS/Ubuntu compatible!
+#
+# This profile contains:
+# - JAVA_HOME - path to currently selected JDK.
+#   jni-rs with enabled the invocation feature fails to build without JAVA_HOME.
+# - JAVA_LIB_DIR - path to libjvm.
+# - RUST_LIB_DIR - path to Rust stdlib.
+# - RUST_VERSION - Rust version for building tests.
+# - RUSTFLAGS - Rust build options augmented with JAVA_LIB_DIR and RUST_LIB_DIR.
+#   That adds paths into search paths where native code at runtime will look for
```

Comment:
absolute paths?

Reply:
On MacOS a path can be relative to the current directory, relative to the current executable, relative to the calling binary (when loading dylib from other executable), relative to @rpath, and absolute.
In this case we add absolute paths into @rpath.
Lets make `RUSTFLAGS=... /wrong_path/Library/Java/JavaVirtualMachines/jdk1.8.0_171.jdk/Contents/Home/jre/lib/server`
```
cargo build --tests

target/debug/deps/integration_tests-1d8668cc32025316
dyld: Library not loaded: @rpath/libjava_bindings.dylib
...
LD_LIBRARY_PATH="target/debug/deps" target/debug/integration_tests-1d8668cc32025316
dyld: Library not loaded: @rpath/libjvm.dylib
...
LD_LIBRARY_PATH="target/debug/deps:/Library/Java/JavaVirtualMachines/jdk1.8.0_171.jdk/Contents/Home/jre/lib/server" target/debug/integration_tests-1d8668cc32025316

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```

Ok, let's look what is in load commands:
```
otool -l target/debug/integration_tests-1d8668cc32025316
...
Load command 14
          cmd LC_LOAD_DYLIB
      cmdsize 56
         name @rpath/libjava_bindings.dylib (offset 24)
   time stamp 2 Thu Jan  1 03:00:02 1970
      current version 0.0.0
compatibility version 0.0.0
...
Load command 16
          cmd LC_LOAD_DYLIB
      cmdsize 48
         name @rpath/libjvm.dylib (offset 24)
   time stamp 2 Thu Jan  1 03:00:02 1970
      current version 1.0.0
compatibility version 1.0.0
...
Load command 22
          cmd LC_RPATH
      cmdsize 96
         path /Users/alexandrsemichenkov/.rustup/toolchains/stable-x86_64-apple-darwin/lib (offset 12)
Load command 23
          cmd LC_RPATH
      cmdsize 96
         path /wrong_path/Library/Java/JavaVirtualMachines/jdk1.8.0_171.jdk/Contents/Home/jre/lib/server (offset 12)
```